### DISCUSSION TITLE
can't visit my github pages showing danger warning

### DISCUSSION THREAD

**Discussion by sharifshehab** [community/community#130938] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

can't visit my github pages showing danger warning

**Comment by zhao-stanley** [comment]
Possible that your site may appear as a dupe of some other website? I've heard of people getting their Pages sites flagged for being very similar to existing services and ending up on the Google Safe Browsing phishing list.

Try looking at the Google Safe Browsing documentation on [Social engineering (phishing and deceptive sites)](https://developers.google.com/search/docs/advanced/security/social-engineering?hl=en).

**Reply by sharifshehab** [reply]
I have deleted those dupe websites from my repository, but it's still showing the danger warning for my other websites in my repository. How can I fix this?
